bert/src/pipeline.rs

```diff
@@ -20,19 +20,17 @@ use thiserror::Error;
 use crate::{
     model::Model,
     pooler::{Embedding1, Embedding2},
-    tokenizer::Tokenizer,
+    tokenizer::Tokenize,
     AveragePooler,
     FirstPooler,
     NonePooler,
 };
 
-/// A pipeline for a bert model.
-///
-/// Can be built from a [`Config`] and consists of a tokenizer, a model and a pooler.
+/// A pipeline can be built from a [`Config`] and consists of a tokenizer, a model and a pooler.
```

Comment:
I think you deleted the short description of what it is  when changing the description of how it can be used.

Reply:
the first line would have been reduced to `A pipeline.` since it is applicable to more than `bert` models now, hence i shortened it a bit.